problemsets-mat223/linearalbegra.tex

```diff
@@ -4927,18 +4973,40 @@ \section*{Determinants}
 		A\mat{1\\0}=\mat{2\\0}\qquad\text{and}\qquad A\mat{0\\1}=\mat{1\\1}.
 	\]
 	\begin{parts}
-	\item Draw $C_2$ and $A(C_2)$, the image of the unit square
+		\item Draw $C_2$ and $A(C_2)$, the image of the unit square
 			under $A$.
-		\item Compute the area of $A (C_2)$.
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item Compute the area of $A(C_2)$.
+			\begin{solution}[inline]
+				The area of this parallelogram is 2.
+			\end{solution}
 		\item Compute $\det(A)$.
+			\begin{solution}
+				$\det(A)=2$. 
+
+				$A=\mat{2&1\\0&1}$, and so $\det(A)=(2)(1)-(1)(0)=2$.
```

Comment:
Change this to: The parallelogram with sides `\mat{2\\1}` and `\math{0\\1}` is positively oriented, so `\det(A)=+2`. In these problems we're emphasizing the sign of the determinant.